README.md

```diff
@@ -1,302 +1,75 @@
+have some special needs it's relatively easy to build and install your own
+customized binaries. More details [here](./SETUP.md)
 
-* It is configured to generate a CloudWatch event, for triggering the Lambda
-  function.
-* The default frequency is every 5 minutes, but it is configurable using
-  CloudFormation
+## Contributing ##
 
-### Lambda function ###
+This project was developed by volunteers in their own spare time, so if it makes
+a difference on your company's bottom line, please consider giving something
+back to ensure further development. You can do it by suggesting improvements,
+contributing some code or documentation, allowing some of your employees to work
+on it for a few hours per week, or even just spreading the word.
 
-* AWS Lambda function connected to the event generator, which triggers it
-  periodically.
-* It has assigned a IAM role and policy with a set of permissions to call the
-  APIs of various AWS services(EC2 and AutoScaling for now) within the user's
-  account.
-* The permissions are the minimal set required for it to work without the need
-  of passing any explicit AWS credentials or access keys.
-* Some algorithm parameters can be configured using Lambda environment
-  variables, based on some of the CloudFormation stack parameters.
-* Contains a handler written in Golang, built using the
-  [eawsy/aws-lambda-go](https://github.com/eawsy/aws-lambda-go) library, which
-  implements a novel aproach that allows Golang code compiled natively to be
-  built in such a way that it can be injected into the Lambda Python runtime.
-* The handler implements all the instance replacement logic.
-  * The spot instances are created by duplicating the configuration of the
-    currently running on-demand instances as closely as possible(IAM roles,
-    security groups, user_data script, etc.) only by adding a spot bid price
-    attribute and eventually changing the instance type to a usually bigger, but
-    compatible one.
-  * The bid price is set to the on-demand price of the instances configured
-    initially on the AutoScaling group.
-  * The new launch configuration may also have a different instance type,
-    determined based on compatibility with the original instance type,
-    considering also how much redundancy we need to have in place in the current
-    availability zone, in order to survive instance termination when outbid for
-    a certain instance type.
+The usual GitHub contribution model applies, but if you would like to raise an
+issue or start working on a pull request, please get in touch on
+[gitter](https://gitter.im/cristim/autospotting) to discuss it first. Any
+random questions are also better asked on gitter.
 
-## Compiling and Installing your own components ##
+Bug reports should contain enough details to be reproduced, see
+[#83](https://github.com/cristim/autospotting/issues/83): for a relatively good
+example:
```

Comment:
Why not use default issue text such as: https://github.com/ansible/ansible/issues/new